Library/Formula/openshift-cli.rb

```diff
@@ -0,0 +1,36 @@
+class OpenshiftCli < Formula
+  desc "OpenShift command-line interface tools"
+  homepage "https://www.openshift.com/"
+  url "https://github.com/openshift/origin.git",
+    :tag => "v1.1.1",
+    :revision => "e1d9873c1d5711b83fd3dd7eefe83a88ceb92c08"
+
+  head "https://github.com/openshift/origin.git"
+
+  depends_on "go" => :build
+
+  def install
+    # this is necessary to avoid having the version marked as dirty
+    ignore_file = buildpath/".git/info/exclude"
+    ignore_file.open("w") do |file|
+      file.puts "/.brew_home"
+    end
+
+    system "make", "all", "WHAT=cmd/openshift", "GOFLAGS=-v"
+
+    arch = MacOS.prefer_64_bit? ? "amd64" : "x86"
+    bin.install "_output/local/bin/darwin/#{arch}/openshift"
+    bin.install_symlink "openshift" => "oc"
+    bin.install_symlink "openshift" => "oadm"
+
+    bash_completion.install "contrib/completions/bash/openshift"
```

Comment:
You can use `bash_completion.install Dir["contrib/completions/bash/*"]` if all the files in this directory should be installed.